presto-native-execution/presto_cpp/main/tests/CMakeLists.txt

```diff
@@ -38,3 +38,17 @@ target_link_libraries(
   gmock
   gtest
   gtest_main)
+
+add_executable(
+  presto_test_server
```

Comment:
hmm, don't know of a good name off the bat but I see someone tripping over the executables presto_test_server and presto_server_test(that's also defined in this file).

Reply:
Thanks for the hint; I've seen `presto_server_test`; that being said, `presto_test_server` appeared to be the most suitable name. If you have a a better name in mind, I'd be happy to use it. (Maybe `presto_test_cpp_server`?)